core/src/main/java/com/linecorp/armeria/client/GoAwayReceivedException.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.client;
+
+/**
+ * A {@link RuntimeException} raised when a server sent an HTTP/2 GOAWAY frame with
```

Comment:
One last nit: How about adding hyperlink to `HTTP/2 GOAWAY frame`?

Reply:
Thanks for addressing that! After I saw your other review about javadoc. I have the same idea.